docs/operations-guide/running-metabase-on-heroku.md

```diff
@@ -45,15 +45,21 @@ Here's each step:
 
 * Clone the latest version to your local machine:
 
+```bash
     git clone https://github.com/metabase/metabase-deploy.git  
     cd metabase-deploy
+```
```

Comment:
This shouldn't stay indented if you add triple backticks, if you look at the rendered page it formats the contents as-is and includes all the extra space on the left margin